test/typing/plain_files/orm/typed_queries.py

```diff
@@ -58,7 +58,7 @@ def t_select_1() -> None:
 
     result = session.execute(stmt)
 
-    # EXPECTED_TYPE: Result[Tuple[int, str]]
+    # EXPECTED_TYPE: Result[int, str]
```

Comment:
well wow.    interesting we get here while still avoiding the problem I was having with p646

Reply:
Thanks to `import as` inside `if TYPE_CHECKING` magic: 
https://github.com/sqlalchemy/sqlalchemy/blob/0352cb931f039cc622618d10d1a7cf2b6ba91aef/lib/sqlalchemy/engine/row.py#L41-L56

Reply:
from what I understand the change is making typechecker thing that row is a tuple subclass, all the rest of the changes are to make sure that it works correctly